src/index.ts

```diff
@@ -53,10 +53,11 @@ const sidecarPlugin: JupyterFrontEndPlugin<void> = {
             });
           } else {
             const anchor = this.model.get('anchor') || 'right';
+            const ref = this.model.get('ref') || null;
             if (anchor === 'right') {
               app.shell.add(w, 'right');
             } else {
-              app.shell.add(w, 'main', { mode: anchor });
+              app.shell.add(w, 'main', { ref: ref, mode: anchor });
             }
             app.shell.activateById(w.id);
```

Comment:
Now that it's created, you can tell the model the view was created

```suggestion
            app.shell.activateById(w.id);
            this.model.resolveCreated();
```